spec/models/game_spec.rb

```diff
@@ -1,11 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Game, type: :model do
-  describe '.new' do
```

Comment:
Can you add this spec code back in? It should still work.

Reply:
Done!